bot/tests/base_test.py

```diff
@@ -36,6 +36,10 @@ def __sanitize_text(text: str) -> str:
         sanitized = " ".join(sanitized.split())
         return sanitized.lower()
 
+    @staticmethod
+    def remove_until_first_space(text: str) -> str:
+        return text.split(' ', 1)[-1] if ' ' in text else text
+
```

Comment:
Nie widziałem wszystkich MRów, ale mam wrażenie że to powinno być częścią TestStart, a nie wypchane do bazowej XD Jak się mylę to resolve

Reply:
Nie pamiętam czy gdzieś to jeszcze użyłem ale patrząc na to jak działa dziwnie to prasowanie markdowna telegramowe to jest to coś co może się przydać w innych teoretycznie xD